live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,6 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
+console.log(str.substr(1,2)); // 'oz'
```

Comment:
Please use the standard "expected output" form, as noted here: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#indicating-consolelog-output

Reply:
Done